apis/objectstorage.k8s.io/v1alpha1/types.go

```diff
@@ -274,6 +278,21 @@ type BucketAccessRequestList struct {
 	Items           []BucketAccessRequest `json:"items"`
 }
 
+type ObjectReference struct {
```

Comment:
Isn't there already a built-in objectreference type?

Reply:
There is, however defining a different ObjectReference type isn't unheard of.  There are a few in base kube.  That said, this one isn't defining anything that the base ObjectReference doesn't have, and in fact has fewer fields.   I'm indifferent to using the existing vs creating our own.

Reply:
[objectreference](https://godoc.org/k8s.io/api/core/v1#ObjectReference)
Your type has fewer fields and I'm fine using it is other project are also defining their own.